### PR TITLE
Feature/cloudwatch

### PR DESCRIPTION
**Pull request by rongwang0930**
1. config cloudwatch for:  es, herd, metastor, bdsql
2. pass release artifact location in cloudformation template

### CODE REVIEW

**Comment by rongwang0930** [comment]
merge to release branch

File: mdl/src/main/cft/installMDL.yml [hunk 1]
```diff
@@ -189,6 +189,10 @@ Parameters:
     Description: Number of Core Instances for the EMR cluster
     Type: String
     Default: '1'
+  ReleaseArtifactLocation:
```
Comment (sbalkee) [inline mdl/src/main/cft/installMDL.yml]: Revisit parameter

File: mdl/src/main/cft/mdlElasticsearch.yml [hunk 2]
```diff
@@ -119,32 +122,28 @@ Resources:
...
-                  - |
                     execute_cmd "./install auto" "RETRY" 2>&1
-                  - |
                     execute_cmd "sudo service codedeploy-agent status"
-                  - |
                     echo 'Code deploy setup complete, sleeping for 2 mins'
-                  - |
                     sleep 2m
+                    execute_cmd "sudo sed -i \"s/{log_group_name}/${logGroupName}/g\" /tmp/logs.conf"
+                    execute_cmd "wget https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py"
```
Comment (sbalkee) [inline mdl/src/main/cft/mdlElasticsearch.yml]: introduce Retry

File: mdl/src/main/cft/mdlElasticsearch.yml [hunk 3]
```diff
@@ -162,6 +161,33 @@ Resources:
               mode: '000644'
               owner: root
               group: root
+            /tmp/logs.conf:
```
Comment (sbalkee) [inline mdl/src/main/cft/mdlElasticsearch.yml]: Revisit the log location

**Pull request merged**